### ISSUE TITLE
About nextTriggerRequest

### ISSUE THREAD

**Issue by ubinWang** [enuren/DVRP#1]
hello, I want to ask a question what is nextTriggerRequest?

**Comment by enuren** [comment]
by nextTriggerRequest I will assume you mean the vaiable in the discrete event simulator?
If this is true we need to go back to how the discrete event simulator functions. It works with a series of events that will be revealed dynamically as time passes in the simulation. That is what the DVRP is all about.
Each time a new event/visit is revealed, the solver will be called so that is can reoptimize the solution. That, however, is not the only new information you get: As time passes, you implicitly know that no visits appeared in the time that passed. This can be important when you for instance want to implement a repositioning strategy.
nextTriggerRequest is a way for the solver to state that it wants to be invoked again at a certain time, even if no further requests  are revealed before then.

TLDR: If the solver is called at time X it can return e.g. (X+10) and if no requests/visits appears before then, the solver will be called again at time (X+10).
